web/app/charts/MapChart.js

```diff
@@ -22,11 +22,14 @@ bluewave.charts.MapChart = function(parent, config) {
             left: 82
         }
     };
-    var svg, mapArea; //d3 elements
+    var svg, mapArea, mapCenter; //d3 elements
     var countyData, countryData; //raw json
     var counties, states, countries; //topojson
     var options = []; //aggregation options
-
+    var projection;
+    var zoomed;
+    var zoom;
```

Comment:
We don't need the zoom variable for the entire class. Remove this and put a var or let or const in front of the zoom in init()

Reply:
will do.